app/main/controllers/template/rtmedia-actions.php

```diff
@@ -969,4 +969,34 @@ function_exists( 'rtmedia_get_album_description_setting' ) && rtmedia_get_album_
 		}
 	}
 }
-add_action( 'rtmedia_gallery_after_title', 'rtmedia_gallery_after_title_callback', 11, 1 );
+add_action( 'rtmedia_gallery_after_title', 'rtmedia_gallery_after_title_callback', 11 );
+
+/**
+ * Show media count in rtMedia gallery shortcode when show_count parameter is set to true..
+ *
+ * @since 4.4.
+ */
+function rtmedia_gallery_after_title_add_media_count() {
+	global $rtmedia_query;
+	// Check for gallery shortcode
+	if ( isset( $rtmedia_query->is_gallery_shortcode ) && true === $rtmedia_query->is_gallery_shortcode ) {
+		global $rtmedia_shortcode_attr;
+		// check in the gallery shortcode $show_count is set to true.
+		if ( isset( $rtmedia_shortcode_attr['show_count'] ) && 'true' == (string) $rtmedia_shortcode_attr['show_count'] ) {
```

Comment:
We need to check `'true' == (string) $rtmedia_shortcode_attr['show_count']` here. because what if any user set `show_count=anything` ?